docs/configure-emulator.md

```diff
@@ -0,0 +1,178 @@
+
+If your shell cannot find the installed binary, make sure `$GOPATH/bin` is in
+the `$PATH` of your shell. Use the following command to add it to your profile
+depending on your shell.
+
+```shell
+export PATH=$HOME/go/bin:$PATH
+echo 'export PATH=$HOME/go/bin:$PATH' >> ~/.profile
+```
+
+## 4. Setting up the covenant emulator program
+
+### 4.1. Initialize directories
+
+Next, we initialize the node and home directory. It should generate all of the 
+necessary files such as `covd.config`, these files will live in the `<path>` 
+that you set for the `--home`with the below command.
+
+```shell
+covd init --home <path>
+```
+
+After initialization, the home directory will have the following structure:
+
+```shell
+$ ls <path>
+  ├── covd.conf # Covd-specific configuration file.
+  ├── logs      # Covd logs
+```
+
+### 4.2. Configure the covenant emulator
+
+As you have already set up the covenant signer, you can now configure the covenant 
+emulator to use it.
+
+Use the following parameters to configure the `covd.conf` file.
+
+```
+# The interval between each query for pending BTC delegations
+QueryInterval = 15s
+
+# The maximum number of delegations that the covd processes each time
+DelegationLimit = 100
+
+# Bitcoin network to run on
+BitcoinNetwork = simnet
+
+# Babylon specific parameters
+
+# Babylon chain ID
+ChainID = bbn-test-5
+
+# Babylon node RPC endpoint
+RPCAddr = https://rpc-euphrates.devnet.babylonlabs.io:443
+
+# Babylon node gRPC endpoint
+GRPCAddr = https://rpc-euphrates.devnet.babylonlabs.io:443
+
+# Name of the key in the keyring to use for signing transactions
+Key = covenant-key
+
+# Type of keyring to use,
+# supported backends - (os|file|kwallet|pass|test|memory)
+# ref https://docs.cosmos.network/v0.46/run-node/keyring.html#available-backends-for-the-keyring
+KeyringBackend = test
+
+[remotesigner]
+; URL of the remote signer
+URL = http://127.0.0.1:9792
+
+; client when making requests to the remote signer
+Timeout = 2s
+
+; if true, covenant will use the remote signer to sign transactions
+RemoteSignerEnabled = true
+```
+
+Ensure that the covenant signer is running and unlocked before proceeding 
+otherwise you will be unable to run the emulator.
+
+## 5. Generate key pairs
+
+The covenant emulator daemon requires the existence of a keyring that signs
+signatures and interacts with Babylon. Use the following command to generate the
+key:
+
+```bash
+$ covd create-key --key-name covenant-key --chain-id chain-test
+{
+    "name": "covenant-key",
+    "public-key": "9bd5baaba3d3fb5a8bcb8c2995c51793e14a1e32f1665cade168f638e3b15538"
+}
+```
+
+After executing the above command, the key name will be saved in the config file
+created in [step](#configuration).
+Note that the `public-key` in the output should be used as one of the inputs of
+the genesis of the Babylon chain.
+Also, this key will be used to pay for the fees due to the daemon submitting 
+signatures to Babylon.
```

Comment:
How can the covenant emulator see the address of their key so that they can receive funds to it?